README.md

```diff
@@ -1,2 +1,344 @@
+
+12.2 You will not without the Fraud Intelligence Limited’s prior written consent make any representations, warranties, guarantees or other commitments with respect to the specifications, features, performance, or capabilities of the Fraud Intelligence Blockchain Service, the Platform, the Fraud Intelligence Data or related services or otherwise incur any liability on behalf of the Fraud Intelligence Limited.
+
+12.3 You will ensure that your employees, agents, and contractors, and any other person to whom You share Fraud Intelligence Data in accordance with the terms of this Agreement comply with the terms of this Agreement. You are responsible for the acts and omissions of each of these persons as if they were acts and omissions of yourself.
+
+12.4 You will: (a) comply with the Policies and Fraud Intelligence Limited’s reasonable directions in respect of the Fraud Intelligence Blockchain Service; (b) cooperate with Fraud Intelligence Limited, Users and other Contributors involved in the Fraud Intelligence Blockchain Service; Fraud Intelligence Blockchain Service Terms of Use 10 (c) promptly notify Fraud Intelligence Limited of any breach of your obligations under this Agreement or any other matter which may impact on your ability to perform those obligations; and (d) not act or omit to act in any way which would or which would reasonably be expected to be considered injurious or detrimental to, to damage or bring into disrepute, Fraud Intelligence Limited Group, its members or Affiliates, other Users or Contributors, or their brands or reputations.
+
+## 13. PERMITTED AFFILIATES
+
+You may disclose to the Permitted Affiliates, and authorize the Permitted Affiliates to use the Fraud Intelligence Blockchain Services and Fraud Intelligence Data, provided that:
+
+13.1 Permitted Affiliates must comply with the restrictions and obligations upon you under this Agreement. Acts and omissions of Permitted Affiliates are deemed to be acts and omissions by you.
+
+13.2 This Agreement does not create a contractual relationship between the Fraud Intelligence Limited and any Permitted Affiliates. Permitted Affiliates shall have no right to enforce any term of, or any rights in relation to, this Agreement.
+
+13.3 An entity may only take the benefit of the provisions of this clause 12 for such period as that entity is your Affiliate.
+
+13.4 If you wish to permit further Affiliates to use the Fraud Intelligence Blockchain Services or Fraud Intelligence Data, you must obtain the prior written consent of the Fraud Intelligence Limited in the form of an amended Order Form. Additional fees may apply. For the avoidance of doubt only Permitted Affiliates named on Your Request to Join may use the Fraud Intelligence Blockchain Service.
+
+13.5 You or your named Permitted Affiliates may apply to Fraud Intelligence Limited for login and API credentials for each Permitted Affiliate, in accordance with this Agreement.
+
+## 13. COMPLIANCE WITH LAWS AND REGULATIONS
```

Comment:
@Mingela I noticed that the number 13 is assigned to two sections (both the current and the previous one are numbered 13). It seems that we need to rectify this, don't we? There are numerous cross-links that will need to be adjusted in this case.

Reply:
I see, let me modify the content fixing the clauses numbering and the very last table. Please also consider moving that from README to an md file named the same way the other two are named.

Reply:
Merged https://github.com/fraud-intelligence-limited/fil-legal/pull/6